content/en/blog/2022/metrics-announcement.md

```diff
@@ -0,0 +1,37 @@
+---
+title: OpenTelemetry Metrics Release Candidates
+date: 2022-05-18
+author: Morgan McLean
+draft: true # TODO: remove this line once your post is ready to be published
+---
+
```

Comment:
Editorial: As a general comment, please consider line-wrapping at 80 chars.

Reply:
Will do on the next one. This one was imported from Google docs and is going live tomorrow, so I'd prefer not to mess with it :)